hyperledger-student-management.yaml

```
@@ -0,0 +1,232 @@
+openapi: "3.0.0"
+info:
+  description: "This is the Student API for UC4's hyperledger component."
+  version: "0.3.3"
+  title: "UC4"
+servers:
+  - url: http://localhost:9000/hyperledger-student-management
+tags:
+- name: "Student Management"
+  description: "Everything about the student"
+  externalDocs:
+    description: "Find out more"
+    url: "https://github.com/upb-uc4"
+paths:
+  /users/students:
+    post:
+      tags:
+      - "Student Management"
+      summary: "Add a new student to the database"
```

database or blockchain?

---

Blockchain. We have not gone over the api-calls yet. So far, we have only adjusted the student to what we think is needed on the blockchain
```suggestion
      summary: "Add a new student to the blockchain"
```